modules/110-istio/docs/README.md

```diff
@@ -64,6 +61,7 @@ The [AuthorizationPolicy](istio-cr.html#authorizationpolicy) resource is respons
 In other words, if you explicitly deny something, then only this restrictive rule will work. On the other hand, if you explicitly allow something, only explicitly authorized requests would be allowed (however, restrictions will have precedence).
 
 You can use the following arguments for defining authorization rules:
+
```

Comment:
@z9r5 What do you think about extra newlines?

Reply:
```shell
root@plodder-master-0:~# kubectl get clusteralerts
NAME               ALERT                                        SEVERITY   AGE     LAST RECEIVED   STATUS
ae9c46e4103fe2a9   D8IstioDeprecatedIstioVersionInstalled       4          3m37s   37s             firing
```

Reply:
Up to you. It doesn't brake markdown linter rules.